guides/common/modules/con_limiting-host-resources.adoc

```diff
@@ -2,6 +2,9 @@
 = Limiting host resources
 
 You can use the Resource Quota plugin to limit users' access to host resources during host provisioning.
+Each host is assigned to exactly one resource quota.
+By default, {Project} assigns existing hosts and hosts during the global host registration workflow to the `Unassigned` resource quota.
```

Comment:
Should this be:
```suggestion
By default, {Project} assigns existing hosts and hosts registered during the global host registration workflow to the `Unassigned` resource quota.
```

Reply:
```suggestion
By default, {Project} assigns hosts registered during the global host registration workflow to the `Unassigned` resource quota.
```

If you deploy a host, you can choose the quota. What you probably mean is that it automatically assigned 'Unassigned' at plugin installation?